portfolio/src/main/java/com/google/sps/data/BasketballCourtsDatastore.java

```diff
@@ -0,0 +1,118 @@
+package com.google.sps.data;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonElement;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableList.Builder;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+import java.util.OptionalInt;
+import java.io.Reader;
+import java.io.InputStreamReader;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import javax.servlet.ServletContext;
+import javax.servlet.*;
+import java.util.Scanner;
+import com.google.sps.data.BasketballCourt;
+
+/** Parses the basketball courts file and returns a list of basketball courts */
+public final class BasketballCourtsDatastore {
+
+  private static final Logger log = Logger.getLogger(BasketballCourtsDatastore.class.getName());
+  
+  public static final String JSON_FILE_PATH = "/WEB-INF/basketball-courts-nyc.json";
+  public static final String ACCESSIBLE_PROPERTY = "Accessible";
+  public static final String IS_ACCESSIBLE = "Y";
+  public static final String ID_PROPERTY = "Prop_ID";
+  public static final String NAME_PROPERTY = "Name";
+  public static final String LOCATION_PROPERTY = "Location";
+  public static final String NUMCOURTS_PROPERTY = "Num_of_Courts";
+  public static final String LAT_PROPERTY = "lat";
+  public static final String LON_PROPERTY = "lon";
+
+  List<BasketballCourt> accessibleCourts;
+  JsonArray allCourts;
+
+  public BasketballCourtsDatastore(ServletContext context) {
+    try(Reader reader = new InputStreamReader(context.getResourceAsStream(JSON_FILE_PATH))) {
+      allCourts = new Gson().fromJson(reader, JsonArray.class);
+      accessibleCourts = BasketballCourtsDatastore.getAccessibleCourtsList(allCourts);
```

Comment:
nit: you don't need the fully qualified class name here, just `getAccessibleCourtsList(allCourts)`